pydantic_ai_slim/pydantic_ai/_cli/__init__.py

```diff
@@ -55,6 +55,11 @@
 
 PROMPT_HISTORY_FILENAME = 'prompt-history.txt'
 
+# CLI-supported tool IDs (excludes deprecated and config-requiring tools)
+_CLI_TOOL_IDS = sorted(
+    k.kind for k in get_builtin_tool_types() if k not in {'mcp_server', 'memory', 'unknown_builtin_tool'}
```

Comment:
I don't think `get_builtin_tool_types` should include `unknown_builtin_tool` at all